packages/golden_toolkit/lib/src/golden_builder.dart

```diff
@@ -99,6 +99,23 @@ class GoldenBuilder {
     ));
   }
 
+  ///  [addScenarioContextWithContext] will add a test with BuildContext GoldenBuilder
+  /// use as:
+  /// ..addScenarioContextWithContext(
+  ///   'Test with context',
+  ///   (context) {
+  ///     return Container();
+  ///   },
+  /// )
+  void addScenarioContextWithContext(
```

Comment:
naming: can we call this addScenarioBuilder() instead?

Reply:
ok